### PR TITLE
[3] Fix bug: correct navbar state management 

### PR DESCRIPTION
**Pull request by dfangshuo**
## Fix bug: correct navbar state management 

Describe your feature here

### Related PRs

1. https://github.com/calblueprint/peoplepower-web/pull/43
2. https://github.com/calblueprint/peoplepower-web/pull/44

### Screenshots

CC: @aivantg @dfangshuo

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -8,6 +8,29 @@
         "extensions": [".js", ".jsx"]
       }
     ],
+    "no-restricted-imports": [
```
Comment (aivantg) [inline .eslintrc.json]: I'd imagine with all these new PRs, there's a lot of shared history between the branches. Ideally to keep it simple as possible, just have the PR include the bugfix but it's not a big deal. I'll just review the relevant stuff for this PR

File: src/components/NavBar.js [hunk 2]
```diff
@@ -1,92 +1,78 @@
...
     return (
       <div className="navBar">
+        <a href="/">
+          <img
+            className="logo"
+            src={Logo}
+            alt="People Power Solar Cooperative Logo"
+          />
+        </a>
+        <nav>
```
Comment (aivantg) [inline src/components/NavBar.js]: We'll actually want a third version of the navbar for if they are currently going through onboarding it doesn't show "Sign in". But that's not crucial

File: src/lib/credentials.js [hunk 3]
```diff
@@ -1,43 +1,56 @@
...
+  }
+
+  if (ownerTypes.includes(GENERAL_OWNER)) {
+    credentials += 'G';
+  }
+
+  return credentials;
+}
+
+function isSignedIn(credentials) {
```
Comment (aivantg) [inline src/lib/credentials.js]: fantastic abstraction barrier. I didn't think of this when Nick and I first wrote the credential code but this is perfect!

File: .eslintrc.json (hunk 1 above)
Comment (dfangshuo) [reply .eslintrc.json]: good idea, will do

**Pull request merged**